### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by ciaranRoche**
## Overview
This change reviews the current documentation, and updates the docs to a number of confirmed ways to deploy and run the operator. 

Changes may satisfy - https://issues.redhat.com/browse/INTLY-4924

### CODE REVIEW

**Comment by aidenkeating** [comment]
:eyes:

**Comment by aidenkeating** [comment]
/lgtm
/approve

this definitely has brought the readme up to date. thanks for doing this

File: README.md [hunk 1]
```diff
@@ -56,8 +56,90 @@ Clean up the Kubernetes/OpenShift cluster:
...
+  namespace: openshift-marketplace
+spec:
+  authorizationToken: {}
+  displayName: Integreatly Operators
+  endpoint: 'https://quay.io/cnr'
+  publisher: Integreatly Publisher
+  registryNamespace: <<your quay org name>>
+  type: appregistry
+```
+Once the OLM has reconciled, the Cloud Resource Operator should be available in the Catalog.
```
Comment (dimitraz) [inline README.md]: You mentioned `CRO` earlier, maybe we can refer to it once as `Cloud resource operator (CRO)` and then as `CRO` afterwards

File: README.md [hunk 2]
```diff
@@ -56,8 +56,90 @@ Clean up the Kubernetes/OpenShift cluster:
...
+      ...
+        - name: cloud-resource-operator
+          image: quay.io/<your quay org>/cloud-resource-operator:v0.7.1
+          command:
+          - cloud-resource-operator
+          imagePullPolicy: Always
+      ...
+```
+
+Seed the Kubernetes/OpenShift cluster with required resources:
```
Comment (dimitraz) [inline README.md]: We could add a Kubernetes/Openshift 4+ cluster as a requirement, and then just refer to this as a cluster onwards

File: README.md [hunk 3]
```diff
@@ -19,14 +19,14 @@ Operator is running.
...
-## Locally
+## Running the Cloud Resource Operator (CRO)
 
-Prerequisites:
+## Prerequisites:
 - `go`
 - `make`
 - [git-secrets](https://github.com/awslabs/git-secrets) - for preventing cloud-provider credentials being included in 
 commits
+- Kubernetes/Openshift 4+ cluster
```
Comment (dimitraz) [inline README.md]:
```suggestion
- Access to an Openshift v4.2.0+ cluster
```

File: README.md [hunk 4]
```diff
@@ -52,12 +53,93 @@ Run the operator:
...
+  registryNamespace: <<your quay org name>>
+  type: appregistry
+```
+Once the OLM has reconciled, the CRO should be available in the Catalog.
+
+*Note* Make sure your Quay Repository and Applications are set to public
+
+## Usage
+### Seeding a Cluster
+After seeding yourcluster with required resources, your cluster is preconfigured to create `managed` and `workshop` resources. Resources deployed in cluster are known as `workshop` and those deployed in AWS are known as `managed`. *Note*, these are arbitrary names and can be updated via the config maps, which are added in the following command :
```
Comment (R-Lawton) [inline README.md]:
```suggestion
After seeding your cluster with required resources, your cluster is preconfigured to create `managed` and `workshop` resources. Resources deployed in cluster are known as `workshop` and those deployed in AWS are known as `managed`. *Note*, these are arbitrary names and can be updated via the config maps, which are added in the following command : 
```

**Pull request closed without merging**